job/job_test.go

```diff
@@ -1028,6 +1108,9 @@ func TestRemoteJobBadStatusSuccess(t *testing.T) {
 	})
 
 	cache := NewMockCache()
+	mockRemoteJob.Init(cache)
```

Comment:
It's curious @tooolbox for this test... with this refactor... need to start cache.. i'll try to dig here, because maybe can throw some light about the problem

Reply:
Sorry, I'm not sure what you're referring to exactly.